kubernetes/samples/scripts/common/utility.sh

```diff
@@ -622,6 +761,47 @@ function createDomain {
   printSummary
 }
 
+#
+# Function to create a domain
+# $1 - boolean value indicating the location of the domain home
+#      true means domain home in image
+#      false means domain home on PV
+#
+function updateDomain {
+  if [ "$#" != 1 ]; then
+    fail "The function must be called with domainHomeInImage parameter."
+  fi
+
+  domainHomeInImage="${1}"
+  if [ "true" != "${domainHomeInImage}" ] && [ "false" != "${domainHomeInImage}" ]; then
+    fail "The value of domainHomeInImage must be true or false: ${domainHomeInImage}"
+  fi
+
+  # Setup the environment for running this script and perform initial validation checks
+  initialize
+
+  # Generate files for creating the domain
+  createFiles update
+
+  # Check that the domain secret exists and contains the required elements
+  validateDomainSecret
+
+  # Validate the domain's persistent volume claim
+  if [ "${doValidation}" == true ] && [ "${domainHomeInImage}" == false -o "${logHomeOnPV}" == true ]; then
```

Comment:
Remove && [ "${domainHomeInImage}" == false -o "${logHomeOnPV}" == true ] since updateDomain is called only by domain-home-on-pv/update-domain.sh

Reply:
fixed